ext/node/ops/util.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+use deno_core::error::AnyError;
+use deno_core::op2;
+use deno_core::OpState;
+use deno_core::ResourceHandle;
+use deno_core::ResourceHandleFd;
+
+#[repr(u32)]
+enum HandleType {
+  #[allow(dead_code)]
```

Comment:
TODO for later work?

Reply:
Dead code on Windows for TCP and UDP is TODO.